src/lib/elements/forms/inputDate.svelte

```diff
@@ -55,45 +54,42 @@
     }
 
     $: isNullable = nullable && !required;
-    $: wrapper = isMultiple ? FormItemPart : FormItem;
 </script>
 
-<svelte:component this={wrapper} {fullWidth}>
-    <Label {required} {optionalText} hide={!showLabel} for={id}>
-        {label}
-    </Label>
+<Label {required} {optionalText} hide={!showLabel} for={id}>
+    {label}
+</Label>
 
-    <div class="input-text-wrapper">
-        <input
-            {id}
-            {disabled}
-            {readonly}
-            {required}
-            step=".001"
-            {min}
-            {max}
-            autocomplete={autocomplete ? 'on' : 'off'}
-            type="date"
-            style={disabled ? '' : 'cursor: pointer;'}
-            class="input-text"
-            bind:value
-            bind:this={element}
-            on:invalid={handleInvalid}
-            on:click={function () {
-                this.showPicker();
-            }}
-            style:--amount-of-buttons={isNullable ? 2.75 : 1}
-            style:--button-size={isNullable ? '2rem' : '1rem'} />
-        {#if isNullable}
-            <ul
-                class="buttons-list u-cross-center u-gap-8 u-position-absolute u-inset-block-start-8 u-inset-block-end-8 u-inset-inline-end-12">
-                <li class="buttons-list-item">
-                    <NullCheckbox checked={value === null} on:change={handleNullChange} />
-                </li>
-            </ul>
-        {/if}
-    </div>
-    {#if error}
-        <Helper type="warning">{error}</Helper>
+<div class="input-text-wrapper">
+    <input
+        {id}
+        {disabled}
+        {readonly}
+        {required}
+        step=".001"
+        {min}
+        {max}
+        autocomplete={autocomplete ? 'on' : 'off'}
+        type="date"
+        style={disabled ? '' : 'cursor: pointer;'}
+        class="input-text"
+        bind:value
+        bind:this={element}
+        on:invalid={handleInvalid}
+        on:click={function () {
+            this.showPicker();
+        }}
+        style:--amount-of-buttons={isNullable ? 2.75 : 1}
+        style:--button-size={isNullable ? '2rem' : '1rem'} />
+    {#if isNullable}
+        <ul
+            class="buttons-list u-cross-center u-gap-8 u-position-absolute u-inset-block-start-8 u-inset-block-end-8 u-inset-inline-end-12">
+            <li class="buttons-list-item">
+                <NullCheckbox checked={value === null} on:change={handleNullChange} />
+            </li>
+        </ul>
     {/if}
-</svelte:component>
+</div>
+{#if error}
+    <Helper type="warning">{error}</Helper>
+{/if}
```

Comment:
I think we need to wrap the component in a `Stack` otherwise the label and the error might be have the wrong gap with the input depending on the parent

Reply:
oh yeah, these should be replaced completely anyway by pinkv2 component